examples/01_forecasting.ipynb

```diff
@@ -5376,7 +5376,8 @@
    "source": [
```

Comment:
<p>we actually dont have a scitype "post-processing transformer", so maybe lets frame that a little bit different?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/3878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='203'/>

Reply:
Thanks!
I've the post-processing from this paragraph and moved it further below, following the other comments.